pkg/controller/nodedaemon/reconcile.go

```diff
@@ -49,13 +49,27 @@ func (r *DaemonReconciler) Reconcile(request reconcile.Request) (reconcile.Resul
 		return reconcile.Result{}, nil
 	}
 
+	lvs, lvTolerations, lvOwnerRefs, lvNodeSeletor, err := r.aggregateLVDaemonInfo(request)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+	if len(lvs.Items) < 1 {
```

Comment:
Now the reconcile won't finish if there aren't both LocalVolumes or LocalVolumeSets.

It should be:

```
if len(lvs.Items) < 1 || len(lvSets.Items) <1 {
    return reconcile.Resut{}, err
}
```
@priyanka19-98 this is why your diskmaker pods aren't coming up

Reply:
What will be the err in
```
if len(lvs.Items) < 1 || len(lvSets.Items) <1 {
    return reconcile.Resut{}, err
}
```
?